test/expected/partials_path_regex.js

```diff
@@ -1,14 +1,15 @@
 this["JST"] = this["JST"] || {};
 
 Handlebars.registerPartial("partial", Handlebars.template({"compiler":[6,">= 2.0.0-beta.1"],"main":function(depth0,helpers,partials,data) {
```

Comment:
@sindresorhus I think testing to check output created by template files doesn't necessary. this PR contains just one updating `package.json` file and many fixes for testing.